src/sentry/testutils/cases.py

```diff
@@ -1232,6 +1235,122 @@ def _send_buckets(cls, buckets, entity):
         )
 
 
+class BaseMetricsLayerTestCase(ABC, BaseMetricsTestCase):
+    @abstractmethod
+    def now(self):
+        """
+        Returns the current time instance that will be used throughout the tests of the metrics layer.
+
+        This method has to be implemented in all the children classes because it serves as a way to standardize
+        access to time.
+        """
+        raise NotImplementedError
+
+    def _store_metric_in_metrics_layer(
```

Comment:
class already has metricsLayer in his name, so you can refactor using just _store_metric